tests/dummy/app/services/gwr-config.js

```diff
@@ -2,7 +2,7 @@ import { action } from "@ember/object";
 import Service from "@ember/service";
 
 export default class GwrConfigService extends Service {
-  cantonAbbreviation = "SZ";
+  cantonAbbreviation = "BE";
```

Comment:
Why the change? :)

Reply:
I refactored the whole canton dev setup a bit. Now you get a prompt on localhost just like the wsk_id.